src/tool-core/models/tool-core/architecture/type/ComponentSpecificType/ComponentSpecificType.xtuml

```diff
@@ -34,6 +34,12 @@ if ( not_empty unresolved_type )
 	unrelate unresolved_type from component across R3805;
 	new_type.updateTypeReferences( old_type:old_type );
 	unrelate old_type from s_dt across R423;
+	select many typeimports related by unresolved_type->Type[R407]->TypeImport[R499.''is imported by''];
```

Comment:
It seems odd for this operation to know this much about type imports.  Perhaps it makes sense to make this an operation on TypeImport?